editor/src/components/inspector/common/property-controls-hooks.ts

```diff
@@ -77,7 +77,10 @@ export function useInspectorInfoForPropertyControl(
 
   const parserFn = unwrapperAndParserForPropertyControl(control)
   const printerFn = printerForPropertyControl(control)
-  const parsedValue = eitherToMaybe(parserFn(rawValues[0], realValues[0])) // TODO We need a way to surface these errors to the users
+  let parsedValue: unknown = null
+  if (0 in rawValues && 0 in realValues) {
```

Comment:
did you hit a new bug with this being unchecked? is this related to the navigator at all?

Reply:
Not related to the navigator, but it was blocking me from testing things.